demo/multi-class.html

```diff
@@ -136,22 +136,20 @@
                 ],
                 "subtasks": subtasks,
                 "initial_line_size": 2,
-                "config_data": {
-                    "default_toolbox_item_order": [
-                        7, // SubmitButtons
-                        0, // ModeSelect
-                        // 1, // ZoomPan
-                        2, // AnnotationResize
-                        3, // AnnotationID
-                        4, // Recolor Active
-                        5, // ClassCounter
-                        6, // KeypointSlider
-                        8, // FilterDistance
-                        // 9, // Brush
-                    ],
-                    "toggle_brush_mode_keybind": "f",
-                    "create_bbox_on_initial_crop": "|",
-                }
+                "default_toolbox_item_order": [
```

Comment:
Can we directly import and use the `enum` here? Instead of using the corresponding `int` val

Reply:
actually i dont think we currently can, as `ulabel.js` doesn't export the enum. We could add that as an export or provide a static method to get it or smth

Reply:
Yeah export that. A list of ints makes this pretty hard to maintain.

Reply:
[c62f946](https://github.com/SenteraLLC/ulabel/pull/170/commits/c62f9462c0acf2ba34594d5f45a1d9285b6d2fd9)